tests/spectron/spellChecker.spectron.js

```diff
@@ -1,28 +1,23 @@
 const Application = require('./spectronSetup');
 const path = require('path');
 const {isMac} = require('../../js/utils/misc.js');
-const childProcess = require('child_process');
 const constants = require('./spectronConstants');
-
+const robot = require('robotjs');
 let app = new Application({});
-let robot;
 
 describe('Tests for spellChecker', () => {
 
     let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
     jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
 
     beforeAll((done) => {
-        childProcess.exec(`npm rebuild robotjs --target=${process.version} --build-from-source`, () => {
-            robot = require('robotjs');
-            app.startApplication().then((startedApp) => {
-                app = startedApp;
-                done();
-            }).catch((err) => {
-                console.error(`Unable to start application error: ${err}`);
-                expect(err).toBeNull();
-                done();
-            });
+        app.startApplication().then((startedApp) => {
+            app = startedApp;
+            done();
+        }).catch((err) => {
+            console.error(`Unable to start application error: ${err}`);
+            expect(err).toBeNull();
```

Comment:
why this? when we are in the catch section, we have to expect err to be not null right?

Reply:
Yes, we are expecting the error to be null so that the test should fail.